packages/devtools-reps/src/object-inspector/tests/component/__snapshots__/getter-setter.js.snap

```diff
@@ -2,22 +2,25 @@
 
 exports[`ObjectInspector - getters & setters renders getters and setters as expected 1`] = `
 "
-▼ x: Getter & Setter
-|  ▶︎ <get>: function x()
-|  ▶︎ <set>: function x()
+▼ root
+|    x: Getter & Setter
+|  ▶︎ <get x()>: function x()
+|  ▶︎ <set x()>: function x()
```

Comment:
why are we `x: Getter & Setter` and the getter setter instead of just the getter and setter?

Reply:
because we want to be able to evaluate `x` at some point :) 
This is an intermediary step, and at some point, instead of `Getter & Setter`, we'll have a button to invoke the getter (and hide setter only properties I think)